src/main/java/com/risevision/bigquery/InsertJobsHandler.java

```diff
@@ -0,0 +1,44 @@
+package com.risevision.bigquery;
+
+import com.google.api.client.extensions.appengine.http.UrlFetchTransport;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class InsertJobsHandler extends HttpServlet {
+  JobInserter inserter;
+  private Logger log = Logger.getAnonymousLogger();
+  private static UrlFetchTransport transport = new UrlFetchTransport();
+
+  public void doGet(HttpServletRequest req, HttpServletResponse resp)
+  throws IOException {
+
+    JobInserterConfig config = ConfigLoader.getJobInserterConfig();
+    String interval = req.getParameter("interval");
+    List<JobInserterConfig.IndividualQueryConfiguration> jobConfigs;
+
+    if(interval == null) {
+      resp.sendError(404);
```

Comment:
@ahmedalsudani If you're going to stick with the parameter requirement that must be 4 or 24 then we should have a better response here.  A 503 with an error message indicating that an `interval` parameter is required might be good.  And the `else` clause below should also state what the error is.